app/src/main.js

```diff
@@ -16,6 +16,13 @@ electronDownload();
 const APP_ARGS_FILE_PATH = path.join(__dirname, '..', 'nativefier.json');
 const appArgs = JSON.parse(fs.readFileSync(APP_ARGS_FILE_PATH, 'utf8'));
 
+if (appArgs.processEnvs) {
+  Object.keys(appArgs.processEnvs).forEach((key) => {
+    // eslint-disable-next-line no-undef
```

Comment:
@githubbob42 I was about to say `.eslintrc`, but our project runs under *two* environments: both node (the builder), and browser (the `app` folder). So, yeah, for now your file-global `eslint-env` looks 👍. Thanks!